lib/primitives/base-element.js

```diff
@@ -29,7 +29,33 @@ module.exports = function(registry) {
      * Creates a deep clone of the instance
      */
     clone: function() {
-      return registry.fromRefract(this.toRefract());
+      var copy = new this.constructor();
+
+      copy.element = this.element;
+
+      if (this.meta.length) {
+        copy._meta = this.meta.clone();
+      }
+
+      if (this.attributes.length) {
+        copy._attributes = this.attributes.clone();
```

Comment:
Yes, since [meta/attributes setter](https://github.com/refractproject/minim/blob/02d426f8585810e79a3e8329bbeccc3a186fabdf/lib/primitives/base-element.js#L293) on BaseElement doesn't replace the ObjectElement but instead passes the given value to `objectElement.set()`. In turn, [`set`](https://github.com/refractproject/minim/blob/02d426f8585810e79a3e8329bbeccc3a186fabdf/lib/primitives/object-element.js#L74) doesn't expect to receive an `ObjectElement` in this way and will fail. You can run the test suite with this line changed to `copy.attributes = this.attributes.clone();` if you want to dig into this further.